docs/DEVELOPING.md

```diff
@@ -343,6 +343,139 @@ $ make ftest NAME=azure_blob_storage
+  - `demo.instance.demo-region.demo.service.com`
+
+##### `port`
+
+The port where the Oracle Database is hosted. Default value is `9090`. Note: For a secured connection, a user needs to configure a SSL port.
+
+  - `5432`
+  - `9090`
+
+##### `username`
+
+The username of the account for the Oracle Database. Default value is `admin`.
+
+##### `password`
+
+The password of the account to be used for the Oracle Database. Default value is `Password_123`.
+
+##### `database`
+
+Name of the Oracle database. Default value is `xe`. Examples:
+
+  - `employee_database`
+  - `customer_database`
+
+##### `tables`
+
+Comma-separated list of tables. The Oracle Database connector will fetch data from all tables present in the configured database, if the value is `*`. Default value is `*`. Examples:
+
+  - `table_1, table_2`
+  - `*`
+
+##### `fetch_size`
+
+Number of rows to fetch on each request to Oracle Database. Default value is `50`.
+
+##### `retry_count`
+
+The number of retry attempts after failed request to Oracle Database. Default value is `3`.
+
+##### `oracle_protocol`
+
+The protocol which the connector uses to establish a connection. Default value is `TCP`. Note: For a secured connection, a user needs to use `TCPS`.
+
+##### `oracle_home`
+
+Path of the Oracle home directory to run connector with thick mode for secured connection. Note: In case of unsecured connection, keep `oracle_home` field empty.
+
+##### `wallet_configuration_path`
+
+Path of the oracle configuration files. Only applicable when configuration files are not at the default location. Note: If Wallet configuration files are located at default location, keep `wallet_configuration_path` field empty.
+
+### Setup for a secured connection with Oracle
+ - User needs to install the Oracle service on the system where the connector is running.
+ - Set the `oracle_home` parameter to the location of your Oracle home directory. If configuration files are not at the default location, set `wallet_configuration_path` parameter.
+  - Now create a directory to store the wallet.
+```shell
+$ mkdir $ORACLE_HOME/ssl_wallet
+```
+  - User needs to create `sqlnet.ora` at `$ORACLE_HOME/network/admin` location and add the following content.
+```
+WALLET_LOCATION = (SOURCE = (METHOD = FILE) (METHOD_DATA = (DIRECTORY = $ORACLE_HOME/ssl_wallet))) 
+SSL_CLIENT_AUTHENTICATION = FALSE 
+SSL_VERSION = 1.0 
+SSL_CIPHER_SUITES = (SSL_RSA_WITH_AES_256_CBC_SHA) 
+SSL_SERVER_DN_MATCH = ON
+```
+  - Then execute the following commands to create a wallet and attach an SSL certificate.
+```shell
+$ orapki wallet create -wallet path-to-oracle-home/ssl_wallet -auto_login_only
+
+$ orapki wallet add -wallet path-to-oracle-home/ssl_wallet -trusted_cert -cert path-to-oracle-home/ssl_wallet/root_ca.pem -auto_login_only
+```
+Replace the file name with your file.
+
+ℹ️ For more information, please refer following [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.Oracle.Options.SSL.html)
```

Comment:
I am confused by the link to AmazonRDS documentation.  Are we saying that the Python connector connects to a secured Oracle server _only via_ AmazonRDS?

Reply:
It's because in this link steps are more accurate and simple.

Reply:
I agree that linking to AmazonRDS docs, without explaining why, is confusing.

Reply:
Added more description about amazon rds link